samples/boards/tlsr9x/timer/src/main.c

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright (c) 2022 Libre Solar Technologies GmbH
```

Comment:
Copyright (c) **2022** **Libre Solar Technologies GmbH**
year and company ?

Reply:
Changed